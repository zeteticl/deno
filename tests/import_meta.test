args: tests/import_meta.ts --reload
output: tests/import_meta.ts.out
