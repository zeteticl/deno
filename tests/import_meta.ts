console.log("import_meta", import.meta.url);

import "import_meta2.ts";
