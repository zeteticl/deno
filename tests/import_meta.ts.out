import_meta2 [WILDCARD]import_meta2.ts
import_meta [WILDCARD]import_meta.ts
